app/Dockerfile

```diff
@@ -0,0 +1,12 @@
+# Docker file to run fastapi and spark
+FROM python:3.8
```

Comment:
```For comparison purposes, the download size of python:3.11-slim-bookworm is 51MB, and python:3.11-alpine3. 18 is 19MB. Their uncompressed on-disk sizes are 149MB and 52MB respectively.```

Reply:
Let's try use python:3.11-alpine. In case we have issues let's upgrade to slim-bookworm. As long as we don't need to install any additional packages.